Makefile

```diff
@@ -2,7 +2,7 @@
 DEIS_REGISTRY ?= $(DEV_REGISTRY)
 IMAGE_PREFIX ?= deis
 COMPONENT ?= workflow
-VERSION ?= git-$(shell git rev-parse --short HEAD)
+VERSION ?= $(shell git rev-parse --short HEAD)
```

Comment:
That would create binaries named: `deis-git-<sha>-<os>-<arch>`

We decided the `git` was not necessary and was actually confusing.